src/map/skill.cpp

```diff
@@ -3231,19 +3231,21 @@ void skill_combo_toggle_inf(struct block_list* bl, uint16 skill_id, int inf){
 		case MO_COMBOFINISH:
 		case CH_TIGERFIST:
 		case CH_CHAINCRUSH:
-			if (sd) clif_skillinfo(sd,MO_EXTREMITYFIST, inf);
+			if (sd != nullptr)
+				clif_skillinfo(*sd,MO_EXTREMITYFIST, inf);
```

Comment:
'inf' from skill_combo_toggle_inf is always 0
```suggestion
				clif_skillinfo(*sd,MO_EXTREMITYFIST);
```